src/pages/api/newspaper/[title_id].ts

```diff
@@ -31,10 +31,12 @@ async function handleGET(titleId: string, box: string, res: NextApiResponse) {
     },
     orderBy: { date: 'asc' }
   }).catch((e: Error) => {
-    if (e instanceof PrismaClientKnownRequestError) { // Error returned from prisma when not found (but request is OK)
-      return res.status(404).json({error: `Failed to find title: ${e.message}`});
+    // PrismaClientKnownRequestError is returned from Prisma when request is OK but something is wrong
+    // P2025 is code for record not found. See https://www.prisma.io/docs/orm/reference/error-reference
+    if (e instanceof PrismaClientKnownRequestError && e.code === 'P2025') {
+      return res.status(404).json({error: `No title found with ID ${titleId}: ${e.message}`});
```

Comment:
det er forventet at en ny eske begynner tomt. Er det da riktig å returnere en 404 når vi finner ingenting i databasen? Og får vi egentlig en P2025 da? Beskrivelsen passer ikke veldig godt med en enkel `findMany` ("An operation failed because it depends on one or more records that were required but not found")

Reply:
Tillegg spørsmål: skal egentlig `getIssuesForTitle` i `local.data.ts` returner 404, eller 200 med tom array?

Reply:
Måtte teste, pr. nå returnerer prisma bare en tom liste, så egentlig er alt det der ganske unødvendig og ikke brukt. Er vell mest rett å returnere tom liste. Så fjerner alt prisma-relatert i error handlingen!

Reply:
Vi bruker jo `findUniqueOrThrow` i `title/[id].ts`, der får vi P2025 feil!

Reply:
Ja, mente bare å fjerne på newspaper-apiet!